Automattic/wp-calypso
# Store: Add action and handlers for fetching product reviews

This PR adds a `fetchReviews` action and related handler code for making a request to the end site  to load product reviews.

This is designed to work with the new endpoint up for review at https://github.com/woocommerce/wc-api-dev/pull/48. That branch is not needed for testing, since this PR currently just ships with tests and changes nothing on the front end.

Reducers are coming in a separate PR to keep the size down.

See #17041.

To Test:
* Load up this branch.
* Run `npm run test-client client/extensions/woocommerce` and make sure all tests pass.
